### PR TITLE
V 021

### PR DESCRIPTION
**Pull request by jvanhoefer**

### CODE REVIEW

**Comment by dweindl** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pypesto/objective/amici_util.py  2
- pypesto/profile/profile.py  1
- test/test_problem.py  4
         

Complexity decreasing per file
==============================
+ pypesto/problem.py  -1
+ pypesto/profile/util.py  -1
+ pypesto/objective/amici_calculator.py  -6
+ pypesto/profile/approximate.py  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=6135592&bid=20063034)

File: doc/releasenotes.rst [hunk 1]
```diff
@@ -5,6 +5,20 @@ Release notes
 0.2 series
 ..........
 
+0.2.1 (2020-09-07)
+------------------
+
+* Example Notebook for prior functionality (#438)
+* changed parameter indexing in profiling routines (#419)
+* basic sanity checking for parameter fixing (#420)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:13]:
```suggestion
* Basic sanity checking for parameter fixing (#420)
```

File: doc/releasenotes.rst [hunk 2]
```diff
@@ -5,6 +5,20 @@ Release notes
 0.2 series
 ..........
 
+0.2.1 (2020-09-07)
+------------------
+
+* Example Notebook for prior functionality (#438)
+* changed parameter indexing in profiling routines (#419)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:12]:
```suggestion
* Changed parameter indexing in profiling routines (#419)
```

File: doc/releasenotes.rst [hunk 3]
```diff
@@ -5,6 +5,20 @@ Release notes
...
+* basic sanity checking for parameter fixing (#420)
+* Bug fixes in:
+  * displaying of multi start optimization (#430)
```
Comment (yannikschaelte) [inline doc/releasenotes.rst:15]:
```suggestion
  * Displaying of multi start optimization (#430)
```

**Review by yannikschaelte — APPROVED** [review]
Can you please change the target to develop branch, and  then from there make a PR named "Release 0.2.1" to master? That is the strategy we always used so far (tbd, if one can do sth better).

**Pull request merged**